# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cups-pdf/cups-group.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./extra/cups-pdf.conf.vanilla	2005-10-04 20:02:57.000000000 +0200
+++ ./extra/cups-pdf.conf	2005-10-04 20:03:52.000000000 +0200
@@ -162,7 +162,7 @@
 ##  created directories and log files
 ### Default: lp
 
-#Grp lp
+Grp root
 
 
 ###########################################################################
